tasks/strings/strings.js

```diff
@@ -4,23 +4,36 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof str != 'string') {
+    return 'This is not a string!';
+  } else {
+    return str.reverse();
```

Comment:
looks like this is not correct, we need to make an array before, reverse it and join, because String does not have reverse() method:
return str.split('').reverse().join('');